### PR TITLE
Add module property config file

### PR DESCRIPTION
**Pull request by llinder**
This follows the config model adopted by the Azure project.

/cc @adriancole @devinsba @aliostad

### CODE REVIEW

File: autoconfigure/collector-sqs/src/main/resources/zipkin-server-sqs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      # Maximum number of messages per receive call
+      max-number-of-messages: ${SQS_MAX_NUMBER_OF_MESSAGES:10}
+      # Optional AWS Access Key
+      aws-access-key-id: ${SQS_AWS_ACCESS_KEY_ID:}
+      # Optional AWS Secret Access Key
+      aws-secret-access-key: ${SQS_AWS_SECRET_ACCESS_KEY:}
+      # Optional AWS STS assume role ARN
+      aws-sts-role-arn: ${SQS_AWS_STS_ROLE_ARN:}
+      # Optional AWS STS region endpoint to use
+      aws-sts-region: ${$SQS_AWS_STS_REGION:us-east-1}
```
Comment (adriancole) [inline autoconfigure/collector-sqs/src/main/resources/zipkin-server-sqs.yml]: typo?

**Comment by adriancole** [comment]
maybe do a copy/paste/find replace on this as a README for collector/sqs?

https://github.com/openzipkin/zipkin-azure/blob/master/collector/eventhub/README.md

File: autoconfigure/collector-sqs/src/main/resources/zipkin-server-sqs.yml (hunk 1 above)
Comment (llinder) [inline autoconfigure/collector-sqs/src/main/resources/zipkin-server-sqs.yml]: good catch.  thank you!

**Comment by llinder** [comment]
Added the much needed documentation.  Based to loosely off the Azure docs though I deviated a bit in splitting things up between the collector and autoconfiguration module.

**Comment by adriancole** [comment]
this is even better than the one I made in azure. fantastic :) I'll use this model in the streaming collector and also bump azure, too.

**Pull request merged**